<launch>

  <!--param name="use_sim_time" value="true" /-->
  <arg name="dataset" /> 

  <node pkg="speed_and_pose_prediction" name="speed_and_pose_prediction_ros_node" type="speed_and_pose_prediction_ros_node" output="screen">
    <param name="odom_topic" value="/odom_localization" />
    <param name="velocity_threshold" value="2.0" />
    <param name="offset_threshold" value="1.0" />
    <param name="acceleration_threshold" value="0.5" />
  </node>
	<!--node pkg="tf" type="static_transform_publisher" name="odom_velodyne_broadcaster" args="0 0 0 0 0 0 /odom /velodyne 100"/-->
</launch>
